### ISSUE TITLE
Discord redirect

### ISSUE THREAD

**Issue by tresf** [LMMS/lmms.io#245]
@lukas-w I find myself sending out a discord invite copy/paste quite a few times a week.  Can we get a clean url that redirects them?  e.g. https://lmms.io/chat and redirects to https://discord.gg/3sc5su7 ?

I looked at https://github.com/LMMS/lmms.io/blob/master/public/index.php and it doesn't seem to have an option for redirects.

This will also have the added benefit of reducing the amount of places we need to update if the invite link needs to be changed.

**Comment by lukas-w** [comment]
@tresf An example for redirecting with Silex can be found [here](https://silex.symfony.com/doc/2.0/usage.html#redirects):
```php
$app->get('/', function () use ($app) {
    return $app->redirect('/hello');
});
```

In our case, it should be as easy as adding the following entry to the `pages` array in `index.php`:

```php
['/chat/', function () use ($app) {
    return $app->redirect('https://discord.gg/3sc5su7');
}]
```

**Comment by tresf** [comment]
Auto-close worked, but the commit didn't crosslink for some strange reason.  Closed via 82a9b3b.
